### DISCUSSION TITLE
Disable/ Enable landing page

### DISCUSSION THREAD

**Discussion by Devarajan89** [colinwilson/lotusdocs#147] [Feature Requests]
If you add the option to disable the landing page it will be easy for who doesn't need the landing page.

**Comment by LITUATUI** [comment]
Agreed!

**Comment by midhunmonachan** [comment]
I do this as a quick workaround

Create `index.html` in `layouts` folder.

```html

<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="refresh" content="0; url=/docs" />
</head>
<body>
</body>
</html>

```


This will automatically redirect it to `/docs`

**Reply by BrainDeadAnarch** [reply]
Thanks for this workaround, I don't have a Layouts folder on a new Hugo\Lotusdocs install. Is there an alternative folder or method please?

EDIT: Ah, figured it out! I needed to create a new Layouts folder. Thanks again.
